<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dials</class>
 <widget class="QDialog" name="dials">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Saft</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>330</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QDial" name="dial_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>211</width>
     <height>181</height>
    </rect>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="dial_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>211</width>
     <height>181</height>
    </rect>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="dial_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>80</y>
     <width>211</width>
     <height>181</height>
    </rect>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Eg f√∏ler meg heldig</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Miks saft</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dials</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dials</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
